t/50reverse-proxy-timings.t

```diff
@@ -68,6 +69,7 @@ run_with_curl($server, sub {
 
     do_upstream($upstream);
     my $resp = join('', <CURL>);
+    diag($resp) if $ENV{TEST_DEBUG};
```

Comment:
I actually wonder if we should have something like below inside Util.pm and call it?
```
sub debug_diag {
    diag($_[0]);
}
```

Reply:
This is going to be helpful; opening #3287.